Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Dockerfile
+
+FROM phusion/passenger-ruby22:0.9.15
+
+# Set correct environment variables.
+ENV HOME /root
+ENV RAILS_ENV production
+
+# Use baseimage-docker's init process.
+CMD ["/sbin/my_init"]
+
+# Expose Nginx HTTP service
+EXPOSE 80
```

Comment:
If there's a way to set up Elastic Beanstalk without an ELB, then I'm all for doing so in this case. For reasons that are probably obvious, the minute you install a load balancer you need to put the keys onto it, and if we're using VPC there's not really a good reason to do the encryption twice. Elastic Beanstalk absolutely supports this case and recommends it http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/configuring-https.html